packages/graphql/src/classes/Neo4jGraphQL.ts

```diff
@@ -128,6 +128,50 @@ class Neo4jGraphQL {
         await assertIndexesAndConstraints({ driver, driverConfig, nodes: this.nodes, options: input.options });
     }
 
+    private wrapResolvers(resolvers: IResolvers, { schema }: { schema: GraphQLSchema }) {
+        const wrapResolverArgs = {
+            driver: this.driver,
+            config: this.config,
+            nodes: this.nodes,
+            relationships: this.relationships,
+            schema,
+        };
+
+        const resolversComposition = {
+            "Query.*": [wrapResolver(wrapResolverArgs)],
+            "Mutation.*": [wrapResolver(wrapResolverArgs)],
+        };
+
+        // this.schemaDefinition.resolvers can either be undefined, IResolvers or IResolvers[]
```

Comment:
**nitpick (non-blocking)**: I think this comment is not too relevant, types should already make that clear